db/migrate/20220422113030_create_shows.rb

```diff
@@ -0,0 +1,19 @@
+class CreateShows < ActiveRecord::Migration[7.0]
+  def change
+    create_table :shows do |t|
+      t.string :show_type, null: false
```

Comment:
you should convert this into integer type because you can then use enums https://betterprogramming.pub/how-to-use-enums-in-rails-6-87600e292476 which is much easier to bulk modify